src/splitInteger.test.js

```diff
@@ -8,12 +8,31 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(splitInteger()).toBeInstanceOf(Array);
+  });
+
+  it(`should return an array with integers`, () => {
+    expect(splitInteger(7, 2)).toEqual([3, 4]);
+  });
 
+  it(`should return an array with length 5 if parts = 5`, () => {
+    expect(splitInteger(30, 5)).toEqual([6, 6, 6, 6, 6]);
   });
```

Comment:
You can also use [toHaveLength](https://jestjs.io/docs/expect#tohavelengthnumber) for this test